backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3263,4 +3263,97 @@ databaseChangeLog:
+  - changeSet:
+      id: add-specimen_type-to-test_order
+      author: zedd@skylight.digital
+      comment: add specimen_type_id to test order
+      changes:
+        - tagDatabase:
+            tag: add-specimen_type-to-test_order
+        - addColumn:
+            tableName: test_order
+            columns:
+              - column:
+                  name: specimen_type_id
+                  afterColumn: device_type_id
+                  remarks: The specimen type for this test order.
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__test_order__specimen_type
+                    references: specimen_type(internal_id)
+  - changeSet:
+      id: add-default_specimen_type_id-to-facility
+      author: zedd@skylight.digital
+      comment: add specimen_type_id to facility
+      changes:
+        - tagDatabase:
+            tag: add-default_specimen_type_id-to-facility
+        - addColumn:
+            tableName: facility
+            columns:
+              - column:
+                  name: default_specimen_type_id
+                  remarks: The default specimen type (if any) for tests at this facility.
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__facility__default_specimen_type
+                    references: specimen_type(internal_id)
+  - changeSet:
+      id: populate-test_event-specimen_type_id
+      author: zedd@skylight.digital
+      changes:
+        - tagDatabase:
+            tag: populate-test_event-specimen_type_id
+        - sql:
+            remarks: Populate test_event.specimen_type_id from test_event.device_specimen_type
+            sql: |
+              update ${database.defaultSchemaName}.test_event
+              set specimen_type_id=(select specimen_type_id from ${database.defaultSchemaName}.device_specimen_type dst where dst.internal_id = device_specimen_type_id)
+              where test_event.device_specimen_type_id is not null and specimen_type_id is null
+  - changeSet:
+      id: populate-test_order-specimen_type_id
+      author: zedd@skylight.digital
+      changes:
+        - tagDatabase:
+            tag: populate-test_event-specimen_type_id
+        - sql:
+            remarks: Populate test_order.specimen_type_id from test_order.device_specimen_type
+            sql: |
+              update ${database.defaultSchemaName}.test_order
+              set specimen_type_id=(select specimen_type_id from ${database.defaultSchemaName}.device_specimen_type dst where dst.internal_id = device_specimen_type_id)
+              where test_order.device_specimen_type_id is not null and specimen_type_id is null
+  - changeSet:
+      id: populate-facility-default_specimen_type_id
+      author: zedd@skylight.digital
+      changes:
+        - tagDatabase:
+            tag: populate-test_event-specimen_type_id
```

Comment:
Same question, should this tag reference `test_event`?

Reply:
good catch, will fix